frontend/src/routes.js

```diff
@@ -11,39 +10,36 @@ const app = express()
 app.set('view engine', 'njk');
 app.use(express.static(path.join(__dirname, "public")));
 
-nunjucks.configure('viewdir', { 
+nunjucks.configure('viewdir', {
     express: app 
 }); 
 
-const api_url = process.env.API_URL;
-
 // Add routes
 app.get('/', async (req, res) => {
-    const response = await fetch(api_url+'api/',{method:'GET',headers:{}})
-    const data = await response.json();
-    res.render('homePage', {test: data}); 
-
+    res.render('homePage'); 
   });
 
+const jobs = require('./jobRoles')
 app.get('/job-roles', async (req, res) => {
-    const response = await fetch(api_url+'job-roles',{method:'GET',headers:{}})
-    const roleData = await response.json();
-    res.render('jobRolesPage', {items: roleData}); 
-
+    res.render('jobRolesPage', {items: await jobs.getJobRoles()}); 
 });
 
 app.get('/job-roles/:id', async (req, res) => {
-    var id = req.params.id;
-    console.log(api_url+'job-roles/'+id)
-    const response = await fetch(api_url+'job-roles/'+id,{method:'GET',headers:{}})
-    const roleData = await response.json();
-    const bandResponse = await fetch(api_url+'bands/'+roleData.bandID,{method:'GET',headers:{}})
-    const bandData = await bandResponse.json();
-    console.log({role: roleData, band: bandData})
-    res.render('jobRolePage', {role: roleData, band: bandData}); 
+    const results = await jobs.getJobRole(req.params.id);
+    res.render('jobRolePage', {role: results[0], band: results[1]});
```

Comment:
If you apply the previous suggestion, this becomes
```suggestion
    res.render('jobRolePage', { role: results.roleData, band: results.bandData });
```